diff -rupEbBN live555-2018.10.17.org/proxyServer/live555ProxyServer.cpp live555-2018.10.17/proxyServer/live555ProxyServer.cpp
--- live555-2018.10.17.org/proxyServer/live555ProxyServer.cpp	2018-12-10 16:16:29.338966465 +1100
+++ live555-2018.10.17/proxyServer/live555ProxyServer.cpp	2018-12-10 16:16:41.448129011 +1100
@@ -271,11 +271,11 @@ int main(int argc, char** argv) {
   // Try first with the default HTTP port (80), and then with the alternative HTTP
   // port numbers (8000 and 8080).
 
-  if (rtspServer->setUpTunnelingOverHTTP(80) || rtspServer->setUpTunnelingOverHTTP(8000) || rtspServer->setUpTunnelingOverHTTP(8080)) {
-    *env << "\n(We use port " << rtspServer->httpServerPortNum() << " for optional RTSP-over-HTTP tunneling.)\n";
-  } else {
-    *env << "\n(RTSP-over-HTTP tunneling is not available.)\n";
-  }
+  //if (rtspServer->setUpTunnelingOverHTTP(80) || rtspServer->setUpTunnelingOverHTTP(8000) || rtspServer->setUpTunnelingOverHTTP(8080)) {
+  //  *env << "\n(We use port " << rtspServer->httpServerPortNum() << " for optional RTSP-over-HTTP tunneling.)\n";
+  //} else {
+  //  *env << "\n(RTSP-over-HTTP tunneling is not available.)\n";
+  //}
 
   // Now, enter the event loop:
   env->taskScheduler().doEventLoop(); // does not return
